app.js

```diff
@@ -0,0 +1,20 @@
+// Packages
+const express = require('express')
+const app = express()
+const PORT = 3000
+
+// JS modules
+// const {openForm,closeForm} = require(__dirname+"/public/js/methods")
+const {openForm, closeForm} = require(__dirname+"/public/js/methods.js")
```

Comment:
Esta linea de código no es necesaria y te produce el error principal "document is not defined" . Simplemente elimínala.